src/features/game/expansion/lib/boosts.ts

```diff
@@ -114,6 +123,13 @@ export const getFoodExpBoost = (
     boostedExp = boostedExp.mul(1.05);
   }
 
+  if (
+    food.name in COOKABLE_CAKES &&
```

Comment:
Now that `COOKABLE_CAKES` exists, can Chef Apron be updated to support Orange Cake?